terraform/helm/aptos-node/values.yaml

```diff
@@ -178,5 +178,14 @@ podSecurityPolicy: true
 # -- Load test-data for starting a test network
 loadTestGenesis: false
 
+# -- TEST ONLY: Enable running as root for profiling
+enablePrivilegedMode: false
```

Comment:
False?

Reply:
It's false by default, but is enabled for Forge separately